.gitmodules

```diff
@@ -1,3 +1,7 @@
 [submodule "deps/feast"]
 	path = deps/feast
 	url = https://github.com/oavdeev/feast
+[submodule "deps/feast-java"]
+	path = deps/feast-java
+	url = https://github.com/feast-dev/feast-java
+	branch = initial_move
```

Comment:
making initial_move the branch now so I can run tests properly but after that branch is merged I'll switch it out